src/domain/login/LoginViewModel.ts

```diff
@@ -15,34 +15,51 @@ limitations under the License.
 */
 
 import {Client} from "../../matrix/Client.js";
-import {ViewModel} from "../ViewModel";
+import {Options as BaseOptions, ViewModel} from "../ViewModel";
 import {PasswordLoginViewModel} from "./PasswordLoginViewModel.js";
 import {StartSSOLoginViewModel} from "./StartSSOLoginViewModel.js";
 import {CompleteSSOLoginViewModel} from "./CompleteSSOLoginViewModel.js";
 import {LoadStatus} from "../../matrix/Client.js";
 import {SessionLoadViewModel} from "../SessionLoadViewModel.js";
 
-export class LoginViewModel extends ViewModel {
-    constructor(options) {
+// TODO(isaiah): make these default exported
```

Comment:
Right, that was more of a note to self and not using proper terminology (although I used it in the commit message, which isn't ideal). What I meant was add them to `index.ts` so that I didn't need to import from each file separately, see

https://github.com/vector-im/hydrogen-web/pull/777/commits/5c5c6007047e10cf0dc00c9d9c5be1b77db99eae